src/components/client-form/ClientForm.jsx

```diff
@@ -81,115 +82,104 @@ function ClientForm({ toggleClientForm, isClientFormVisible, createClient }) {
       <div className={styles.personalDetailsTab}>Client details</div>
       <form className={styles.form} onSubmit={handleSubmit} ref={formRef}>
         <div className={styles.formColumnOne}>
-          <label className={styles.labelName} htmlFor="clientName">
+          <span className={styles.labelName} htmlFor="clientName">
```

Comment:
Why did you replace the `label` with a `span`?

Reply:
because linting had issues with label missing a control. Probably because a regular html input was missing